README.rst

```diff
@@ -39,13 +39,15 @@ To run the tests::
 Translations
 ------------
 
-To update translations from Transifex, `pip install transifex-client` and run::
+To update translations from Transifex, run::
 
     make txpull
     python manage.py compilemessages
 
 To push new strings to Transifex, run::
 
+    python manage.py makemessages -l en
+    # set "nplurals" and "plural" in djangopeople/locale/en/LC_MESSAGES/django.po
```

Comment:
I'm not sure. Django's source `.po` files don't have a `Plural-Forms` entry but when I ran makemessages I found a stub plural-forms header. I thought it'd be useful to set this for correct plural rules but maybe there is a default/implicit rule that's safe for English (nplurals: 2, plural: n!=1).

I'll remove this and squash with first commit.